Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
```

Comment:
quick note, you dont need this import css file because your already importing it in a higher order component.